lib/linguist/heuristics.yml

```diff
@@ -400,12 +400,12 @@ disambiguations:
     pattern: "(?i:(alter module)|(language sql)|(begin( NOT)+ atomic)|signal SQLSTATE '[0-9]+')"
   # Oracle
   - language: PLSQL
-    pattern: '(?i:\$\$PLSQL_|XMLTYPE|sysdate|systimestamp|\.nextval|connect by|AUTHID (DEFINER|CURRENT_USER)|constructor\W+function)'
+    pattern: '(?i:\$\$PLSQL_|XMLTYPE|sysdate|systimestamp|\.nextval|connect by|AUTHID (DEFINER|CURRENT_USER)|constructor\W+function|FROM DUAL)'
```

Comment:
The test failures are legitimate and suggest `FROM DUAL` is _not_ unique to PLSQL and can be used for "normal" SQL too. From a quick search, this appears correct as MySQL allows the use of it too: https://dev.mysql.com/doc/refman/8.0/en/select.html

Reply:
Agree with this.